### ISSUE TITLE
Equalizer support?

### ISSUE THREAD

**Issue by drcryogen** [ejurgensen/forked-daapd#289]
Does forked-daapd have EQ support when sending to AirPlay devices like iTunes does? 

Maybe I'm just missing how to do this but I'd like to be able to, for example, reduce bass to the remote speakers so as to not irritate the neighbors during playback.

**Comment by ejurgensen** [comment]
No, there is no support for that.  It would be possible to add support for this using an [ffmpeg filter](https://ffmpeg.org/ffmpeg-filters.html#equalizer), but then the equalizer would apply to all speakers.  I'm not sure if that would be good enough? Having an individual equalizer for each speaker wouldn't be so easy to support.

**Comment by drcryogen** [comment]
I was indeed just thinking a single EQ applied to all speakers, similar to how iTunes does it. What you suggest would definitely be good enough for what I was looking for.

I can imagine it would probably be really annoying to listen to multiple speakers with individual EQ settings. Especially if the settings were drastically different on different speakers. 

---------------------------
Message created using 100% recycled electrons

**Comment by MatejLach** [comment]
I'd also be happy with one EQ settings for all speakers, if this were to be implemented. 

Thanks for your hard work @ejurgensen

**Comment by sebasjunca** [comment]
Hi all!

Any news on how to implement the EQ? Or how to use an ffmpeg filter to achieve it?

@ejurgensen thanks for forked-daapd, it's amazing!
